.gitignore

```diff
@@ -1,2 +1,6 @@
+<<<<<<< HEAD
```

Comment:
You forgot to resolve a merge conflict here.

Reply:
Fixed!

Reply:
And as for the testcase png files, those are the ones being generated by the code so I think it would be good to leave them be

Reply:
Actually some of the figures were outdated so i've removed those too